app/styles/main.scss

```diff
@@ -252,6 +251,37 @@ body {
         }
     }
 
+    .db-page-header {
```

Comment:
BTW I started using `db-` prefix instead of `ad-` just to avoid potential adblocker conflicts.